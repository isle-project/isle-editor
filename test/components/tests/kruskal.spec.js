// MODULES //

import React from 'react';
import renderer from 'react-test-renderer';
import '@testing-library/jest-dom';
import Kruskal from '@isle-project/components/tests/kruskal';
import heartdisease from '../fixtures/heartdisease.json';


// TESTS //

describe( '<Kruskal />', function test() {
	it( 'renders test result', () => {
		const output = renderer.create( <Kruskal
			data={heartdisease}
			variable="Cost"
			group="Gender"
		/> );
		expect( output ).toMatchSnapshot();
	});
});
