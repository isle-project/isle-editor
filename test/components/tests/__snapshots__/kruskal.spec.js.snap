// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Kruskal /> renders test result 1`] = `
<div
  style={
    {
      "overflowX": "auto",
      "width": "100%",
    }
  }
>
  <label>
    kruskal-wallis-test
  </label>
  <pre
    style={
      {
        "marginTop": 10,
      }
    }
  >
    Kruskal-Wallis Test

Null hypothesis: the medians of all groups are the same

    pValue: 0.7501
    statistic: 0.1015    df: 1


  </pre>
</div>
`;
